lib/graphiti/sideload/belongs_to.rb

```diff
@@ -48,6 +48,16 @@ def child_map(children)
   end
 
   def children_for(parent, map)
-    map[parent.send(foreign_key)]
+    fk = parent.send(foreign_key)
```

Comment:
This is complex enough logic shared between the two locations (with pk vs fk) that it would be nice to extract to util method/class